### ISSUE TITLE
Fails to fetch extension config if the version starts with v  

### ISSUE THREAD

**Issue by 404passionFruit** [oat-sa/tao-release-notes#8]
**TL;DR** if some of your **extensions versions start with the letter v**, they **won't be included in all_notes.md**

This issue was found during preparation for deployment to preprod

```
Could not resolve tao-community vv0.149.1-alpha
❎ invalid json response body at https://raw.githubusercontent.com/oat-sa/tao-community/vv0.149.1-alpha/composer.json reason: Unexpected token : in JSON at position 3
FetchError: invalid json response body at https://raw.githubusercontent.com/oat-sa/tao-community/vv0.149.1-alpha/composer.json reason: Unexpected token : in JSON at position 3
    at /home/sv/git/tao-release-notes/node_modules/node-fetch/lib/index.js:272:32
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async Object.resolveTaoCommunityComposer (/home/sv/git/tao-release-notes/src/notes/requests.js:60:34)
    at async expandTaoCommunity (/home/sv/git/tao-release-notes/src/releaseNotes.js:95:36)
    at async findAllKeys (/home/sv/git/tao-release-notes/src/releaseNotes.js:76:13)
    at async buildExtensionRanges (/home/sv/git/tao-release-notes/src/releaseNotes.js:49:22)
    at async downloadMultiple (/home/sv/git/tao-release-notes/src/releaseNotes.js:169:29)
    at async wrap (/home/sv/git/tao-release-notes/src/releaseNotes.js:189:9) {
  type: 'invalid-json'
}
 ✅ Retrieved 0 extension versions for tao-community vv0.149.1-alpha.
➡ Looking up tao-community vv0.150.1-alpha on github
Could not resolve tao-community vv0.150.1-alpha
❎ invalid json response body at https://raw.githubusercontent.com/oat-sa/tao-community/vv0.150.1-alpha/composer.json reason: Unexpected token : in JSON at position 3
FetchError: invalid json response body at https://raw.githubusercontent.com/oat-sa/tao-community/vv0.150.1-alpha/composer.json reason: Unexpected token : in JSON at position 3
    at /home/sv/git/tao-release-notes/node_modules/node-fetch/lib/index.js:272:32
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async Object.resolveTaoCommunityComposer (/home/sv/git/tao-release-notes/src/notes/requests.js:60:34)
    at async expandTaoCommunity (/home/sv/git/tao-release-notes/src/releaseNotes.js:95:36)
    at async findAllKeys (/home/sv/git/tao-release-notes/src/releaseNotes.js:77:13)
    at async buildExtensionRanges (/home/sv/git/tao-release-notes/src/releaseNotes.js:49:22)
    at async downloadMultiple (/home/sv/git/tao-release-notes/src/releaseNotes.js:169:29)
    at async wrap (/home/sv/git/tao-release-notes/src/releaseNotes.js:189:9) {
  type: 'invalid-json'
  ```
  So to make it work I had to temporarily remove `v` from composer files. Example:
 ` "oat-sa/tao-community": "0.150.1-alpha"` instead of `"oat-sa/tao-community": "v0.150.1-alpha"`
 
 Would be nice to make it work properly out of the box, because this issue doesn't break the execution and logs for other extensions appear after the error. This makes it easy to miss and we could have partial notes in PRs if we didn't pay close attention to all the logs from `taoReleaseNotes`

**Comment by oatymart** [comment]
Closed by https://github.com/oat-sa/tao-release-notes/releases/tag/v1.1.0
